lib/index.coffee

```diff
@@ -55,4 +55,38 @@ class DockerGC
 			console.log('GC: Done.')
 			return _.every(results)
 
+	getDaemonFreeSpace: () ->
```

Comment:
You should probably document somewhere that the result is in kilobytes.

Reply:
Or return it in bytes.

Reply:
Good point!